### PR TITLE
docs: bump minimum supported version of APM Server

### PR DESCRIPTION
**Pull request by watson**
Depends on ~~elastic/docs#457~~

### CODE REVIEW

File: docs/supported-technologies.asciidoc [hunk 1]
```diff
@@ -28,7 +28,9 @@ that not all core Node.js API's can be instrumented without the use of Async Hoo
 [[elastic-stack-compatibility]]
 === Elastic Stack Compatibility
 
-This agent is compatible with {apm-server-ref}[APM Server] v6.2 and above.
+This agent is compatible with {apm-server-ref}[APM Server] v6.5 and above.
+For support for previous releases of the APM Server,
+use version {apm-node-ref-1x}[1.x] of the agent.
```
Comment (watson) [inline docs/supported-technologies.asciidoc]: I'm surprised this didn't break the docs build 🤔

Comment (bmorelli25) [reply docs/supported-technologies.asciidoc]: Are you surprised because it didn't break locally?

Comment (watson) [reply docs/supported-technologies.asciidoc]: I would have thought it would check if `apm-node-ref-1x` was a valid attribute when doing the docs build

Comment (bmorelli25) [reply docs/supported-technologies.asciidoc]: Link checking only happens when you build _all_ of the documentation. Which is what I learned when I broke the docs for six hours yesterday :(

**Pull request merged**